sdk/servicebus/Azure.Messaging.ServiceBus/Azure.Messaging.ServiceBus.sln

```diff
@@ -28,6 +28,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{8B8C
 		samples\Sample06_Transactions.md = samples\Sample06_Transactions.md
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.Experimental", "..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj", "{5871D9C6-F2DF-4F05-B29A-6C0D8709784A}"
```

Comment:
Do we want a project reference for this or should we just be linking as shared source?   I would assume that anything from experimental that gets promoted would flow into the `Azure.Core` package that we already reference.   Not sure if it makes sense to include an `Azure.Core.Experimental.dll` as part of our preview package.    Probably not a big deal, but linking to the source would eliminate the need to distribute the experimental DLL.

Reply:
I think it is okay to distribute the experimental DLL, at least as far as my understanding goes for what @pakrym mentioned the plan was.